src/App.js

```diff
@@ -15,7 +14,8 @@ class App extends Component {
     this.auth = new AuthService()
     this.state={
       authenticated: this.auth.loggedIn(),
-      user: {}
+      user: {},
+      userId: (this.auth.loggedIn()) ? this.auth.getUserId() : undefined
```

Comment:
using `undefined` here is a code smell.  Try null or blank string.  Whatever makes more sense as a default value